components/CollapsibleList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { View, Text, FlatList, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import fonts from '../styles/fonts';
+import colours from '../styles/colours';
+
+const itemStyle = StyleSheet.create({
+  container: {
+    paddingLeft: 30,
+    paddingVertical: 10,
+    flex: 1,
+    flexDirection: 'column',
+    borderBottomWidth: 1,
+    borderColor: colours.black,
+  },
+  text: {
+    fontSize: fonts.normal,
+    color: colours.black,
+  },
+});
+
+const renderItem = ({ item }) => (
+  <View style={itemStyle.container}>
+    <Text style={itemStyle.text}>{'Latitude: '}{item.latitude.toFixed(5)}</Text>
+    <Text style={itemStyle.text}>{'Longitude: '}{item.longitude.toFixed(5)}</Text>
```

Comment:
🔴 these terms need to be languified. Should clicking on the items show them on the map? We  could also show a mini map.

Reply:
You might want to pull this out to a separate component to isolate the styling and let us modify it on it's own.

Reply:
Yeah, my plan was to make them go to the actual pin on the map if possible. Probably do that in a separate pr

Reply:
Clicking a failed item will now bring you to the OverviewMap screen and centre the map on the failed location pin. (In the future, we could show the pin callout right away as well using refs)